### DISCUSSION TITLE
The search functionnality in the doc site seems not working correctly

### DISCUSSION THREAD

**Discussion by erdnaxeli** [renovatebot/renovate#17902] [Suggest an Idea]
Hi,

As said in the title, I have trouble with the search functionality in the doc site. For exemple there is an entry for "onboardingConfig":
![image](https://user-images.githubusercontent.com/479441/191467935-fdb60226-df36-4518-b6ce-389730d6cd6f.png)

But searching "onboardingConfig" does not return this result:
![image](https://user-images.githubusercontent.com/479441/191468058-c7a61eda-9c3d-4189-ba9b-68d61affb7ea.png)

**Comment by viceice** [comment]
you need to click the `<x> more in this page` link

**Reply by erdnaxeli** [reply]
Argh, thank you. But why the exact word match is not the first proposed answer? This is confusing.

**Reply by viceice** [reply]
Don't know seems to be a mkdocs issue 🤷‍♂️
